config/config.go

```diff
@@ -57,6 +62,7 @@ func Get() Config {
 	_ = viper.BindEnv("log.level", "LOG_LEVEL")
 	_ = viper.BindEnv("api.key", "API_KEY")
 	_ = viper.BindEnv("api.url", "API_URL")
```

Comment:
No it does not read any files.
The content (text,bytes etc) of the CA must be supplied [like in the agent here](https://github.com/castai/k8s-agent/pull/134/files#diff-6cfc2f815064fb7851e423891d1a9bd434f1ec8de8949143019178e5d75e433dR105).
I didn't like the word `File` as part of the parameter name but I decided to keep it because the agent has it that way (even though it has nothing to do with "files").

Reply:
I changed my mind. I will delete `_FILE` suffix because it is misleading.

Reply:
Oh sadly I have to keep the `_FILE` suffix because the customer will usually reference the secret that they already use for agent which has `data.TLS_CA_CERT_FILE` 😞